indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/helpers/WalletHelper.kt

```diff
@@ -0,0 +1,78 @@
+package com.luxoft.blockchainlab.hyperledger.indy.helpers
+
+import com.luxoft.blockchainlab.hyperledger.indy.utils.SerializationUtils
+import com.luxoft.blockchainlab.hyperledger.indy.utils.getRootCause
+import org.hyperledger.indy.sdk.wallet.Wallet
+import org.hyperledger.indy.sdk.wallet.WalletAlreadyOpenedException
+import org.hyperledger.indy.sdk.wallet.WalletExistsException
+import java.util.concurrent.ExecutionException
+
+/**
+ * Helps to manage wallets
+ */
+object WalletHelper {
```

Comment:
I think API similar to Pool is better, 
create() < throw exception if exist
openExists() < throw exception if doesn't exist
openOrCreate() < handle exception and create new wallet

Reply:
Ok.